functions-python/preprocessed_analytics/function_config.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "process-analytics",
+  "description": "Process analytics",
+  "entry_point": "process_analytics",
```

Comment:
I'm curious, and this is really minor, but why is the folder named `preprocessed_analytics` (past tense) while the entry point is process_analytics (present tense without the `pre`)

Reply:
Great question! The `function_config` only allows for one entry point, but in reality, this source code creates two different Cloud Functions. If you check the Terraform script, you'll notice that `gtfs` or `gbfs` is added as a suffix to each function to handle the differences between them.

Reply:
So the entry point in function_config is not really use as-is, right?